nuxeo-operation.html

```diff
@@ -21,7 +21,7 @@
     <nuxeo-operation auto
         op="Document.Query"
         params="{'query': 'select from Document'}"
```

Comment:
Since we're updating documentation, there is a `*` missing here. It should be `select * from Document`.